### PR TITLE
added: readme file

### PR DESCRIPTION
**Pull request by simplify-apps**
## Check yourself

- [ ] Code is linted
- [ ] Tested
- [ ] All new/updated code is covered with tests

## Security

- [ ] Security impact of change has been considered
- [ ] Code follows security practices and guidelines

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,160 @@
...
+```
+yarn add simplify-redux-app
+```
+
+then create module
+
+```jsx
+import { simplifyBuilder } from 'simplify-redux-app';
+
+// create builder that will handle all works
```
Comment (vozmarkov) [inline README.md]: // create builder that will handle all the work

File: README.md [hunk 2]
```diff
@@ -1,2 +1,160 @@
...
+const builder = simplifyBuilder(state, existReducers);
+
+// create reducer for that will be used by redux
```
Comment (vozmarkov) [inline README.md]: // create reducer that will be used by redux

File: README.md [hunk 3]
```diff
@@ -1,2 +1,160 @@
...
+// example how to create simple redux action
+const addTodoItem = builder.createReduxAction((test: string) => ({
+  name: 'ADD_TODO_ITEM',
+  updater: (state) => ({
+    ...state,
+    items: [...state.items, test],
+  }),
+}));
+
+// example how to send request a server
```
Comment (vozmarkov) [inline README.md]: // example how to send a server request

File: README.md [hunk 4]
```diff
@@ -1,2 +1,160 @@
...
+```
+
+for the simple redux store update you can use `createReduxAction`, for example `addTodoItem` will create promise
+```typescript
+const addTodoItem: (test: string) => Promise<SimpleAction<ITodoState, [test: string]>>
+```
+
+and simply by dispatching it your react component `dispatch(addTodoItem('test'))` your redux store will be updated, that's it!
+
+in the case when you need to send the request to the server via HTTP and put the result into redux, you can use `createServerAction`. For example `loadTodoItemById` will will create promise:
```
Comment (vozmarkov) [inline README.md]: double will, remove one 'will'

File: README.md [hunk 5]
```diff
@@ -1,2 +1,160 @@
...
+
+```typescript
+const loadTodoItemById: (id: string) => Promise<SimpleAction<ITodoState, APITodoItemResponse>>
+```
+
+and like previously by dispatching the function in the to react component your store will be updated!
```
Comment (vozmarkov) [inline README.md]: and like previously by dispatching the function in the react component your store will be updated!

File: README.md [hunk 6]
```diff
@@ -1,2 +1,160 @@
...
+
+if you will call `toString` method on any function, that was created from SRA build, it will return the `name` of the action. (no more constants 😅)
+
+Wanna more? Check the `example` folder or ask questions
```
Comment (vozmarkov) [inline README.md]: Want more

**Review by vozmarkov — COMMENTED** [review]
overall looks good and easy to follow. :)

File: README.md (hunk 1 above)
Comment (simplify-apps) [reply README.md]: thx

File: README.md (hunk 2 above)
Comment (simplify-apps) [reply README.md]: thx

File: README.md (hunk 3 above)
Comment (simplify-apps) [reply README.md]: thx

File: README.md (hunk 4 above)
Comment (simplify-apps) [reply README.md]: thx

**Pull request merged**